content/docs/accessibility.md

```diff
@@ -104,81 +102,82 @@ function ListItem({ item }) {
+Daha fazla bilgi için, [Fragmentler dokümantasyonu](/docs/fragments.html)'na bakınız.
 
-## Accessible Forms {#accessible-forms}
+## Erişilebilir Formlar {#accessible-forms}
 
-### Labeling {#labeling}
-Every HTML form control, such as `<input>` and `<textarea>`, needs to be labeled accessibly. We need to provide descriptive labels that are also exposed to screen readers.
+### Etiketlemek {#labeling}
+`<input>` ve `<textarea>` gibi her HTML form kontrolunun (form control), erişilebilir halde etiketlenmiş olması gerekir. Ekran okuyucuları tarafndan da ortaya çıkan, açıklayıcı etiketler sağlamamız gerekir.
 
-The following resources show us how to do this:
+Aşağıdaki kaynaklar bize bunun nasıl yapılacağını gösterir:
 
-- [The W3C shows us how to label elements](https://www.w3.org/WAI/tutorials/forms/labels/)
-- [WebAIM shows us how to label elements](https://webaim.org/techniques/forms/controls)
-- [The Paciello Group explains accessible names](https://www.paciellogroup.com/blog/2017/04/what-is-an-accessible-name/)
+- [W3C bize öğelerin nasıl etiketleneceğini gösterir](https://www.w3.org/WAI/tutorials/forms/labels/)
+- [WebAIM bize öğelerin nasıl etiketleneceğini gösterir](https://webaim.org/techniques/forms/controls)
+- [Paciello Group erişilebilir isimleri açıklar](https://www.paciellogroup.com/blog/2017/04/what-is-an-accessible-name/)
 
-Although these standard HTML practices can be directly used in React, note that the `for` attribute is written as `htmlFor` in JSX:
+Bu standart HTML uygulamaları doğrudan React'te kullanılabilmesine rağmen, `for` niteliğinin JSX'de `htmlFor` olarak yazıldığına dikkat edin:
 
 ```javascript{1}
 <label htmlFor="namedInput">Name:</label>
 <input id="namedInput" type="text" name="name"/>
 ```
 
-### Notifying the user of errors {#notifying-the-user-of-errors}
+### Kullanıcı hatalarını bildirmek {#notifying-the-user-of-errors}
 
-Error situations need to be understood by all users. The following link shows us how to expose error texts to screen readers as well:
+Hata durumlarının tüm kullanıcılar tarafından anlaşılmış olması gerekir. 
+Aşağıdaki link, hata metinlerinin de ekran okuyucuları tarafından nasıl algılandığını gösterir:
 
-- [The W3C demonstrates user notifications](https://www.w3.org/WAI/tutorials/forms/notifications/)
-- [WebAIM looks at form validation](https://webaim.org/techniques/formvalidation/)
+- [W3C, kullanıcı bildirimlerini gösterir](https://www.w3.org/WAI/tutorials/forms/notifications/)
+- [WebAIM, form doğrulamaya bakar](https://webaim.org/techniques/formvalidation/)
 
-## Focus Control {#focus-control}
+## Odak Kontrolü {#focus-control}
 
-Ensure that your web application can be fully operated with the keyboard only:
+Web uygulamanızın yalnızca klavye ile tam olarak çalıştırılabildiğinden emin olun:
 
-- [WebAIM talks about keyboard accessibility](https://webaim.org/techniques/keyboard/)
+- [WebAIM, klavye erişilebilirliği hakkında konuşur](https://webaim.org/techniques/keyboard/)
```

Comment:
what about `konusuna deginir` or `bahseder` ?

Reply:
verb of 'deginir' is like to touch something lightly
verb of 'bahseder' sounds like giving a brief explanation.
verb of 'konusur' is the exact Turkish translation to 'talks' within the title.
If  you are not satisfied with the exact translation, you are free to replace it with the choice of yours.